### PR TITLE
[V0.1.2] Rename core & init packages.

### PR DESCRIPTION
**Pull request by Heyrows**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] All new and existing tests are passing.

### CODE REVIEW

**Review by Miniplop — APPROVED** [review]

File: packages/slice-machine/package.json [hunk 1]
```diff
@@ -38,8 +38,7 @@
   "dependencies": {
     "@babel/register": "^7.11.5",
     "@reach/menu-button": "^0.10.5",
-    "@slicemachine/core": "^1.0.0",
```
Comment (Miniplop) [inline packages/slice-machine/package.json]: weird O.O

Comment (Heyrows) [reply packages/slice-machine/package.json]: indeed looool, I just did a make clean and make lol

File: packages/slice-machine/package.json [hunk 2]
```diff
@@ -38,8 +38,7 @@
...
-    "@svgr/webpack": "^5.4.0",
+    "@slicemachine/core": "^1.0.0",    "@svgr/webpack": "^5.4.0",
```
Comment (Heyrows) [inline packages/slice-machine/package.json]: just some kind of linter issue i guess

Comment (Miniplop) [reply packages/slice-machine/package.json]: can you put this dependency at a new line like all the others :)

**Pull request merged**